classes/QuickAccess.php

```diff
@@ -104,7 +104,7 @@ public static function getQuickAccessesWithToken($idLang, $idEmployee)
 
                     $tokenString = $admin_tab[1] . (int) Tab::getIdFromClassName($admin_tab[1]) . $idEmployee;
                 }
-                $quickAccess[$index]['link'] = '/' . basename(_PS_ADMIN_DIR_) . '/' . $quick['link'];
+                $quickAccess[$index]['link'] = Context::getContext()->shop->physical_uri . basename(_PS_ADMIN_DIR_) . '/' . $quick['link'];
```

Comment:
No, this won't work. You can't use $context->shop as it is not related to the URL used to access the admin area.